src/seedu/addressbook/AddressBook.java

```diff
@@ -211,6 +211,7 @@ public static void main(String[] args) {
         processProgramArgs(args);
         loadDataFromStorage();
         while (true) {
+        	echoEnterCommand();
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.